test/import_translation_test.py

```diff
@@ -0,0 +1,44 @@
+import subprocess
+import json
+import os
+from glob import glob
+from dace import SDFG
+# TODO: Change to LIT tests
+folderPath = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Yesn't
LIT tests are great for correctness tests. We basically provide a input/output pair and do a one to one check. This way we can make sure that the translator actually outputs what we expect. However this is labour-intensive since even fairly short SDIR code can result in a long SDFG JSON. This test just checks if DaCe accepts the produced SDFG, which couples the test result to the DaCe implementation (so bugs in DaCe might propegate to this test) and furthermore it does not check if the produced SDFG is what is expected. For example a translator that just outputs an empty string would pass this test for any input.

So I should probably change the comment to `Add LIT tests` and it would also be neat if we could start this test with CMake. What do you think?

Reply:
I think there's no point in making LIT tests for SDFG JSONs. Their formats may slightly change over time as well so comparisons wouldn't work at some point. Instead, you can provide equivalent programs (or if you want, minimal SDFG files) to diff with, or even run a script that checks for correctness.

Reply:
Agreed